src/components/common/animations/Error/data.json

```diff
@@ -1,27 +1,13 @@
 {
-  "v": "5.5.7",
+  "v": "5.6.2",
   "fr": 30,
   "ip": 0,
-  "op": 88,
-  "w": 97,
-  "h": 129,
+  "op": 45,
+  "w": 93,
+  "h": 93,
   "nm": "Error_X",
   "ddd": 0,
   "assets": [],
-  "fonts": {
-    "list": [
-      {
-        "origin": 0,
-        "fPath": "",
-        "fClass": "",
-        "fFamily": "Roboto Slab",
```

Comment:
I can suppose that Roboto Slab references in the invalid data.json are the reason of the crashes happened.
Probably there are no such font on target Android system, but is was selected in AfterEffects.
An if to export correctly it fallbacks to some system font